### PR TITLE
This will add missing itunes fields such as categories and keywords

### PR DESCRIPTION
**Pull request by sanathks**
This will add missing itunes fields such as categories and keywords

expected output: 
```
"itunes" {
...
 "categories": [
        "Society & Culture",
        "Games & Hobbies"
      ],
 "keywords": [
        "Culture",
        "Society"
      ],
}
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "chai": "^3.4.1",
     "express": "^4.16.3",
     "mocha": "^5.2.0",
-    "puppeteer": "^1.8.0",
+    "puppeteer": "^1.16.0",
```
Comment (sanathks) [inline package.json]: Had to do a `npm audit fix ` , due to 2 high severity vulnerabilities (found by npm audit)

**Comment by sanathks** [comment]
@bobby-brennan it's ready to be reviewed.

File: lib/parser.js [hunk 2]
```diff
@@ -222,6 +222,8 @@ class Parser {
   decorateItunes(feed, channel) {
     let items = channel.item || [],
         entry = {};
+    let categories = [];
+    let keywords = [];
```
Comment (bobby-brennan) [inline lib/parser.js]: Looks like `keywords` is unused

**Comment by bobby-brennan** [comment]
Thanks for the contribution! Aside from the one comment, looks good to me.

File: lib/parser.js (hunk 2 above)
Comment (sanathks) [reply lib/parser.js]: My Bad, I removed it.

**Comment by sanathks** [comment]
It's ready again

**Comment by bobby-brennan** [comment]
Thanks!

**Comment by bobby-brennan** [comment]
Published as `3.7.1`

**Pull request merged**